src/tokentype.js

```diff
@@ -109,6 +109,7 @@ export const types = {
   starstar: new TokenType("**", {beforeExpr: true}),
 
   // Keyword token types.
+  _await: kw("await"),
```

Comment:
> **await** is only treated as a *FutureReservedWord* when Module is the goal symbol of the syntactic grammar.
> http://www.ecma-international.org/ecma-262/7.0/#sec-future-reserved-words

`await` is not always keyword. But it's always keyword in Module. (The spec says "The following tokens are reserved for used as keywords in future language extensions." and async/await is the future language extension.)

So I wrote that it switches to keywords at https://github.com/ternjs/acorn/pull/446/files#diff-230fff7c0b3ed8f0d4f2443a246fc76aR17